### PR TITLE
Catch exception when creating a new module

### PR DESCRIPTION
**Pull request by blinkseb**
Display a log message telling which module was being constructed when the exception occurred. Can be useful for debugging.

Note: based on #103. I'll rebase once merged

### CODE REVIEW

File: examples/tt_fullyleptonic.lua [hunk 1]
```diff
@@ -240,7 +240,7 @@ Looper.looper = {
       x_end = 2000.,
       n_bins = 500,
 
-      ps_weight = 'cuba::ps_weight',
+      ps_weight = 'cuba:ps_weight',
```
Comment (swertz) [inline examples/tt_fullyleptonic.lua]: That shouldn't change ;)

Comment (blinkseb) [inline examples/tt_fullyleptonic.lua]: :blush: indeed

Comment (blinkseb) [inline examples/tt_fullyleptonic.lua]: And fixed

**Comment by blinkseb** [comment]
Rebase :+1:

**Pull request merged**